### ISSUE TITLE
OParl Export

### ISSUE THREAD

**Issue by konstin** [meine-stadt-transparent/meine-stadt-transparent#35]
Ist zwar im momentanen Status nicht relevant, steht aber in der Featureliste drinnen.

**Comment by akuckartz** [comment]
Nach "Important" nun "invalid" ?! Gibt es das Feature nun oder nicht?

**Comment by CatoTH** [comment]
Nein, bis Ende Februar wird es das nicht geben, da wir bislang nur Daten verarbeiten, die bereits aus einer OParl-Schnittstelle stammen, und bei denen ein OParl-Export daher redundant wäre. Der wird erst relevant, wenn wir auch Daten aus einer anderen Quelle (anderer Standard oder Scraper) beziehen.
